### PR TITLE
Merge releases/2021/3

### PR DESCRIPTION
**Pull request by ntyukaev**

### CODE REVIEW

**Review by andrew-zaytsev — APPROVED** [review]

File: demos/3d_segmentation_demo/python/README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Options:
 ```
 
 Running the application with the empty list of options yields the usage message and an error message.
-To run the demo, use public or pre-trained models that support 3D convolution, for example, UNet3D. You can download the pre-trained models using the OpenVINO [Model Downloader](../../../tools/downloader/README.md). The list of models supported by the demo is in [models.lst](./models.lst).
+To run the demo, use public or pre-trained models that support 3D convolution, for example, UNet3D. You can download the pre-trained models using the OpenVINO [Model Downloader](../../../tools/downloader/README.md). The list of models supported by the demo is in the `models.lst` file in the demo's directory.
```
Comment (IRDonch) [inline demos/3d_segmentation_demo/python/README.md:59]:
```suggestion
To run the demo, use public or pre-trained models that support 3D convolution, for example, UNet3D. You can download the pre-trained models using the OpenVINO [Model Downloader](../../../tools/downloader/README.md). The list of models supported by the demo is in `<omz_dir>/demos/3d_segmentation_demo/python/models.lst`.
```

This would be more direct, as well as more robust, because I plan to implement automatic checking for `<omz_dir>` references.

File: models/public/resnet-50-tf/README.md [hunk 2]
```diff
@@ -92,4 +92,4 @@ python3 <omz_dir>/tools/downloader/converter.py --name <model_name>
 
 The original model is distributed under the
 [Apache License, Version 2.0](https://raw.githubusercontent.com/tensorflow/models/master/LICENSE).
-A copy of the license is provided in [APACHE-2.0-TF-Models.txt](../licenses/APACHE-2.0-TF-Models.txt).
+A copy of the license is provided in [APACHE-2.0-TensorFlow.txt](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/licenses/APACHE-2.0-TF-Models.txt).
```
Comment (IRDonch) [inline models/public/resnet-50-tf/README.md:95]: Please use an `<omz_dir>` reference instead.

File: models/public/retinanet-tf/README.md [hunk 3]
```diff
@@ -10,24 +10,23 @@ converted to TensorFlow\* protobuf format. For details, see [paper](https://arxi
...
+   git checkout c841508a88faa5aa1ffc7a4947c3809ea4ec1228
+   ```
+   3. Apply `keras_to_tensorflow.patch`:
+   ```
+   git apply keras_to_tensorflow.patch
+   ```
+   4. Run script:
+   ```
+   python keras_to_tensorflow.py --input_model=<model_in>.h5 --output_model=<model_out>.pb
+   ```
```
Comment (IRDonch) [inline models/public/retinanet-tf/README.md:29]: This change makes the output look worse. Previously the code blocks were within the list items; now they're on the same level as the list items.

I also don't understand why you changed the indentation from 4 spaces to 3.

File: tools/accuracy_checker/configs/README.md [hunk 4]
```diff
@@ -55,7 +55,7 @@ Example:
...
-4. The path relative to which the `data_source` is specified can be provided via  `-s, --source` command line. If you want to evaluate models using well-known datasets, you need to organize folders with validation datasets in a certain way. More detailed information about dataset preparation you can find in [Dataset Preparation Guide](../../../data/datasets.md). In order, if you want use data source in quantization via Post Training Optimization Toolkit, you should use environment variable `DATA_DIR` for specifying path to root of directories with datasets.
+4. The path relative to which the `data_source` is specified can be provided via  `-s, --source` command line. If you want to evaluate models using well-known datasets, you need to organize folders with validation datasets in a certain way. More detailed information about dataset preparation you can find in [Dataset Preparation Guide](https://github.com/openvinotoolkit/open_model_zoo/blob/release/data/datasets.md). In order, if you want use data source in quantization via Post Training Optimization Toolkit, you should use environment variable `DATA_DIR` for specifying path to root of directories with datasets.
```
Comment (IRDonch) [inline tools/accuracy_checker/configs/README.md:58]: I just changed this to a relative link; why did you change it back?

**Review by IRDonch — CHANGES_REQUESTED** [review]

File: models/public/retinanet-tf/README.md (hunk 3 above)
Comment (andrew-zaytsev) [reply models/public/retinanet-tf/README.md:29]: It's all about how Doxygen works... If we move the code block to the subitems, the formatting is broken in output

**Review by andrew-zaytsev — APPROVED** [review]

**Review by andrew-zaytsev — APPROVED** [review]

File: models/public/resnet-50-tf/README.md [hunk 5]
```diff
@@ -92,4 +92,4 @@ python3 <omz_dir>/tools/downloader/converter.py --name <model_name>
 
 The original model is distributed under the
 [Apache License, Version 2.0](https://raw.githubusercontent.com/tensorflow/models/master/LICENSE).
-A copy of the license is provided in [APACHE-2.0-TF-Models.txt](../licenses/APACHE-2.0-TF-Models.txt).
+A copy of the license is provided in `<omz_dir>/models/public/licenses/APACHE-2.0-TensorFlow.txt`.
```
Comment (IRDonch) [inline models/public/resnet-50-tf/README.md:95]: Why did you change the link target?

Comment (ntyukaev) [reply models/public/resnet-50-tf/README.md]: fixed

File: tools/accuracy_checker/configs/README.md (hunk 4 above)
Comment (IRDonch) [reply tools/accuracy_checker/configs/README.md]: You didn't answer.

Comment (ntyukaev) [reply tools/accuracy_checker/configs/README.md]: fixed

File: models/public/retinanet-tf/README.md (hunk 3 above)
Comment (IRDonch) [reply models/public/retinanet-tf/README.md:29]: OK. I'm guessing you're referring to doxygen/doxygen#6241.

**Review by IRDonch — APPROVED** [review]
Mainly looks good, but please fix the errors from the trigger build.

**Comment by andrew-zaytsev** [comment]
Errors from trigger build are fixed. Could you please merge?
